examples/crf-listener.c

```diff
@@ -385,7 +386,7 @@ static bool is_valid_crf_pdu(struct avtp_crf_pdu *pdu)
 		return false;
 	}
 	if (val64 != CRF_DATA_LEN) {
-		fprintf(stderr, "CRF Data length mismatch: expected %lu, got %lu\n",
+		fprintf(stderr, "CRF Data length mismatch: expected %zd, got %" PRIu64 "\n",
```

Comment:
CRF_DATA_LEN type is `size_t`, right? So it is an unsigned type and we should use `%zu` instead of `%zd`.

Reply:
@aguedes - if I look here https://github.com/AVnu/libavtp/blob/25f1bc1ed4e3f629fe044b89fde3a4fb278f6eb7/examples/crf-listener.c#L115 I can see that CRF_DATA_LEN is really just the number 48 at compile time, so I don't think the `size-t` comment applies. In fact, I think the usage of `%z` is incorrect in this instance.

Reply:
The sizeof operator yields a `size_t` type which is implementation-defined. For instance, gcc defines it as `long unsigned int` for 64-bit and `unsigned int` for 32-bit (x86). So my understanding is that the `%z` is needed here indeed.

Reply:
There is no sizeof() operation in the coe that I am reading https://github.com/AVnu/libavtp/blob/master/examples/crf-listener.c#L388
?

The parameter passed in to %zd is a #define.

Can you please identify the sizeof() operation you are referring to?

Reply:
The CRF_DATA_LEN macro is expanded to an expression where the `sizeof` operation is used.

Reply:
@aguedes - duh - you are correct. My bad.